### DISCUSSION TITLE
Welcome to GPT3Discord Discussions!

### DISCUSSION THREAD

**Discussion by Kav-K** [Kav-K/GPTDiscord#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and what you use this project for, and any other thoughts!

**Comment by fr8nki3** [comment]
Thanks for all the work!

**Comment by Kav-K** [comment]
@fr8nki3 my pleasure!
